### PR TITLE
Mailchimptosendyform

### PR DESCRIPTION
**Pull request by scbadkas19**
This is an edit to the blockchained India website where the subscribe button in the footer redirects to sendy form and not to mailchimp form.
The format used for the html form is same as that of mailchimp.
This is done so that from now on all our subscriber data is fed into sendy form.

### CODE REVIEW

File: server/views/components/footer.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <a href="https://blockchainedindia.github.io/resources" target="_blank">Resources</a>
         <a href="/talks">Previous Talks</a>
     </div>
-    <form class="column subscribe" action="https://blockchainedindia.us15.list-manage.com/subscribe/post?u=43f23a03fbc4cf7179d9d1800&amp;id=de00b92d69" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
+    <form class="column subscribe" action="https://github.com/scbadkas19/website/blob/mailchimptosendyform/server/views/components/sendyform/subscribeform_sendy.html" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
```
Comment (anistark) [inline server/views/components/footer.html:18]: Local routes doesn't work like this.

You need to add the route on nodejs express code and refer url route here.

File: server/views/components/sendyform/subscribeform_sendy.html [hunk 2]
```diff
@@ -0,0 +1,898 @@
+
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Blockchained India</title>
```
Comment (anistark) [inline server/views/components/sendyform/subscribeform_sendy.html:5]: Title should be more descriptive.

**Review by anistark — CHANGES_REQUESTED** [review]
Also, share a screenshot of how the page looks on your local system.

File: server/views/components/sendyform/subscribeform_sendy.html [hunk 3]
```diff
@@ -0,0 +1,898 @@
...
+			width:65px;
+			float:left;
+			margin-right:15px;
+		}
+</style></head>
+
+
+	<body>
+		<div class="wrapper rounded6" id="templateContainer">
+		    <h1 class="masthead"><img src="https://github.com/scbadkas19/website/blob/mailchimptosendyform/server/views/components/sendyform/BIsendyformimage.png" alt="" border="0" style="border:0px  ; border-color:; border-style:; border-width: 0px; height: 500px; width: 1500px; margin: 0; padding: 0;" width="1500" height="500"></h1>
```
Comment (anistark) [inline server/views/components/sendyform/subscribeform_sendy.html:850]: Also change any absolute links to either `blockchainedindia` repo or partial url route to static resources.

Comment (anistark) [inline server/views/components/sendyform/subscribeform_sendy.html:850]: Either change all absolute links to `blockchainedindia` repo links or change to relative urls or call from assets static resources.

File: server/views/components/sendyform/subscribeform_sendy.html [hunk 4]
```diff
@@ -0,0 +1,898 @@
...
+						<input type="text" name="name" id="name"/>
+					</div>
+					
+					<div>
+						<label for="email">Email</label>
+						<input type="email" name="email" id="email"/>
+					</div>
+					
+					<div><label for="City">City</label><input type="text" name="City" id="City"/></div>					
+					<input type="hidden" name="list" value="6UTl6xuM2ILTiYE7njQ72A"/>
```
Comment (anistark) [inline server/views/components/sendyform/subscribeform_sendy.html:870]: Is this a secret key? If so, don't commit it to github. Call from backend env and call config from backend.

**Comment by scbadkas19** [comment]
> Also, share a screenshot of how the page looks on your local system.

> Also, share a screenshot of how the page looks on your local system.

https://github.com/scbadkas19/website/blob/mailchimptosendyform/server/views/components/sendyform/ss.png

**Comment by anistark** [comment]
> > Also, share a screenshot of how the page looks on your local system.
> 
> > Also, share a screenshot of how the page looks on your local system.
> 
> https://github.com/scbadkas19/website/blob/mailchimptosendyform/server/views/components/sendyform/ss.png

1. Please resolve the above as commented.
2. The logo is old logo. Please update with latest logo.

File: server/views/components/sendyform/subscribeform_sendy.html (hunk 4 above)
Comment (chinmayshah99) [reply server/views/components/sendyform/subscribeform_sendy.html:870]: This is not a secret key.

**Comment by anistark** [comment]
@scbadkas19 This is still not working. Please test thoroughly and fix the issue. Open a new PR. This one is merged and deployed.

**Pull request merged**